community-spotlight-conversational-search.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Distilling news articles and legal research into simple chat experiences'
+description: 'Putting the community spotlight on Agustín Gomez and Alvaro Machuca, two developers based in Paraguay building real world applications for conversational search with your data.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/min-database-indexing.png
+  alt: Community Spotlight
+author: Alex Francoeur
+date: 10-12-2023
+published: true
+slug: community-spotlight-chat-search-experiences
+---
+
+Today we’re putting the community spotlight on two developers based out of Paraguay, [Agustín Gomez](https://github.com/gomezag) and (Alvaro Machuca)\[https://github.com/quanturtle]. They discovered Xata while looking for a serverless database offering that supported vector embeddings. Since starting with Xata a few months ago, Agustín and Alvaro have built two practical applications to provide chat experiences on very specific sets of data. Both apps have a similar tech stack and build experiences on top of Python / Django, Xata, and Vercel.
```

Comment:
```suggestion
Today we’re putting the community spotlight on two developers based out of Paraguay, [Agustín Gomez](https://github.com/gomezag) and (Alvaro Machuca)[https://github.com/quanturtle]. They discovered Xata while looking for a serverless database offering that supported vector embeddings. Since starting with Xata a few months ago, Agustín and Alvaro have built two practical applications to provide chat experiences on very specific sets of data. Both apps have a similar tech stack and build experiences on top of Python / Django, Xata, and Vercel.
```